Jellyfin.Versioning/update-version

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+# Jellyfin.Versioning/update-version
+# Part of the Jellyfin project (https://jellyfin.media)
+#
+#    All copyright belongs to the Jellyfin contributors; a full list can
+#    be found in the file CONTRIBUTORS.md
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 2 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+if [ -d ../.git ]; then
+    commit=`git rev-parse HEAD`
+    count=`git rev-list HEAD --count`
+    branch=`git rev-parse --abbrev-ref HEAD`
+    desc=`git describe --tags --always --long`
+    remote=`git config --get remote.origin.url`
+    echo commit=$commi\nrevision=$count\nbranch=$branch\ntagdesc=$desc\nremote=$remote > jellyfin_version.ini
+    echo Updated build version in jellyfin_version.ini
+    echo commit=$commit
+    echo revision=$count
+    echo branch=$branch
+    echo tagdesc=$desc
+    echo remote=$remote
+else
+    echo Did not update build version because there was no .git directory.
```

Comment:
Good point. Is it easy to make the caller ignore the exit code? I really want to have it exit with a non-zero code to make it clearer to users of this script that it failed.

Reply:
I suppose I could add either color to the message or allcapps `ERROR: `. The trouble is dotnet build will not ignore a non-zero exit code. And since this is a script that users should never really call themselves.

It is really only an error if the file didn't already exist, but then the build is just fail.